fvm/bootstrap.go

```diff
@@ -89,6 +98,13 @@ func WithMinimumStorageReservation(reservation cadence.UFix64) BootstrapProcedur
 	}
 }
 
+func WithStoragePerFlow(ratio cadence.UFix64) BootstrapProcedureOption {
```

Comment:
I would add the units to be explicit. Also could be good to say `FLOW` to indicate that it's referring to the token.

```suggestion
func WithStorageMBPerFLOW(ratio cadence.UFix64) BootstrapProcedureOption {
```

Reply:
fixed in 926ad3502784e8b51269df72f6f4ad89d698296d